sdk/servicebus/azure-servicebus/azure/servicebus/aio/_base_handler_async.py

```diff
@@ -147,9 +147,14 @@ def __init__(
         else:
             self._credential = credential # type: ignore
         self._container_id = CONTAINER_PREFIX + str(uuid.uuid4())[:8]
+        # overriding default 0.6 for async, since, in async, longer socket timeout interval
+        # doesn't slow down receives when waiting on a socket to return for empty queue/entity.
+        # Chose arbitrary value of 1 since it works.
```

Comment:
This is documented as 0.6, so we should not just arbitrarily ignore that. Has async been stress tested at 0.6?

Reply:
We have not stress tested the default value, but we can.

Reply:
updated to 0.2